stable/bitbucket/templates/bitbucket-statefulset.yaml

```diff
@@ -0,0 +1,139 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ template "bitbucket.fullname" . }}
+  labels:
+    app: {{ template "bitbucket.name" . }}
+    chart: {{ template "bitbucket.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  serviceName: {{ template "bitbucket.fullname" . }}
+{{ if .Values.serviceAccountName }}
```

Comment:
Please see https://github.com/helm/helm/blob/master/docs/chart_best_practices/rbac.md for howto do rbac/serviceaccount.